indexer/block_indexer.go

```diff
@@ -67,6 +67,14 @@ func NewBlockIndexer(
 	}
 }
 
+func (bi *BlockIndexer) AddNewAddressesOfInterest(addresses ...string) {
```

Comment:
Are we sure we just want to add and not to remove old ones in all the cases?
I mean, maybe this function should have recreation of map too

```
bi.addressesOfInterest = make(map[string]bool, len(addresses))
````

If it is too complicated for apex-bridge to pass both new and old address, leave this function as it is